src/main/java/codeu/model/data/Bot.java

```diff
@@ -0,0 +1,43 @@
+package codeu.model.data;
```

Comment:
Since we're planning to have a profile page for the bot, how about making this extend the `User` class? It's reasonable to think of this as a specialized type of user, and we get the `id`, `name`, `creation`, and `aboutMe` (and maybe even `isAdmin`?) fields for free, which all make sense to me to have on a `Bot` instance. This way we also don't have to handle it differently on the chat JSP.

Reply:
I hadn't thought of this, thanks! For this approach, would it still make sense to assign the same `id` on each `Bot` instance?

Reply:
Not really, bots are different entities, much like different users, so they'd get a unique id each in this case.